### ISSUE TITLE
1.2.1 regression: test_schema_resource_access fails

### ISSUE THREAD

**Issue by mgorny** [sissaschool/xmlschema#197]
With all Pythons I have, the following test fails in 1.2.1:

```pytb
$ tox -e py38
GLOB sdist-make: /tmp/xmlschema-1.2.1/setup.py
py38 inst-nodeps: /home/mgorny/.tox/xmlschema/.tmp/package/1/xmlschema-1.2.1.zip
py38 installed: elementpath==1.4.5,lxml==4.5.1,memory-profiler==0.57.0,psutil==5.7.0,xmlschema @ file:///home/mgorny/.tox/xmlschema/.tmp/package/1/xmlschema-1.2.1.zip
py38 run-test-pre: PYTHONHASHSEED='3143484311'
py38 run-test: commands[0] | python -m unittest
..........................................s..............................................................F..........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
======================================================================
FAIL: test_schema_resource_access (tests.test_resources.TestResources)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/xmlschema-1.2.1/tests/test_resources.py", line 788, in test_schema_resource_access
    self.assertEqual(len(ctx), 1, "Expected one import warning")
AssertionError: 0 != 1 : Expected one import warning

----------------------------------------------------------------------
Ran 644 tests in 57.217s

FAILED (failures=1, skipped=1)
ERROR: InvocationError for command /home/mgorny/.tox/xmlschema/py38/bin/python -m unittest (exited with code 1)
_____________________________________________________________________ summary _____________________________________________________________________
ERROR:   py38: commands failed
```

**Comment by brunato** [comment]
This may happens if the tests were run under _/tmp_ directory.
Anyway I will change _/tmp_ with an improbable path in the next release.

Thanks

**Comment by brunato** [comment]
Test fix is available with v1.2.2.
